### PR TITLE
Do not fail if messaging bus could not be started

### PR DESCRIPTION
**Pull request by simao**

### CODE REVIEW

File: common-messaging/src/main/scala/org/genivi/sota/messaging/nats/NatsClient.scala [hunk 1]
```diff
@@ -26,22 +26,23 @@ import scala.util.Try
...
+  private[this] def fromConfig(system: ActorSystem, config: Config): Throwable Xor Conn = {
     for {
       natsConfig <- config.configAt("messaging.nats")
-      userName   <- natsConfig.readString("user")
-      password   <- natsConfig.readString("password")
-      host       <- natsConfig.readString("host")
-      port       <- natsConfig.readInt("port")
+      userName <- natsConfig.readString("user")
+      password <- natsConfig.readString("password")
+      host <- natsConfig.readString("host")
```
Comment (CalumMcCall) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/nats/NatsClient.scala]: This spacing is inconsistent with the rest of the file.

Comment (simao) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/nats/NatsClient.scala]: the rest of the file is wrong.

Comment (simao) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/nats/NatsClient.scala]: added a new commit to fix it :)

**Comment by CalumMcCall** [comment]
11/11

**Pull request merged**